### DISCUSSION TITLE
If, how, and where are gene name synonyms/gene identifyers handled when using gp.prerank() ?

### DISCUSSION THREAD

**Discussion by RobStrasser** [zqfang/GSEApy#266] [Q&A] [answered]
Hi! 

While trying to look at all genes (i.e. not only leading-edge genes, but really all genes under that term in the datebase) from the KEGG terms that were in my gp.prerank() result, I noticed that plenty of genes were present in my data under a different name, e.g. "MT-ND1" instead of "ND1". Could someone tell me how this discrepancy is resolved within gseapy? That would help a lot to see if there are preprocessing steps I'd need to do before submitting my ranked genes.

**Comment by zqfang** [comment, marked as answer]
gseapy will not resolve the gene alias or synonyms for you. By default, the genes in the Enrichr database are all official gene symbols.

You may utilize your own gene identifiers, provided that your input data and gene sets use the same type of gene identifier.
